pyscitt/pyscitt/client.py

```diff
@@ -153,6 +153,7 @@ def request(
         url,
         *,
         retry_on=[],
+        wait_time=None,
```

Comment:
What about making this a property of the Client instead? You can easily create a "modified" client in tests by taking the `client` fixture and calling `client.replace(wait_time=0)`.

See the authentication tests, where it gets used to set credentials: https://github.com/microsoft/scitt-ccf-ledger/blob/566de22714d6ebc7745a8582a617da2ef5d832a8/test/test_auth.py#L50